be/src/olap/rowset/beta_rowset_writer.cpp

```diff
@@ -748,12 +801,15 @@ Status BetaRowsetWriter::build(RowsetSharedPtr& rowset) {
                                                                   : _context.tablet_schema;
     _rowset_meta->set_tablet_schema(rowset_schema);
 
-    if (auto idx_files_info = _idx_files_info.get_inverted_files_info(_segment_start_id);
-        !idx_files_info.has_value()) [[unlikely]] {
-        LOG(ERROR) << "expected inverted index files info, but none presents: "
-                   << idx_files_info.error();
-    } else {
-        _rowset_meta->add_inverted_index_files_info(idx_files_info.value());
+    // If segment compaction occurs, the idx file info will become inaccurate.
```

Comment:
How to handle the idx_files_info in segcompaction?

Reply:
Don't handle it, the index info within the rowset meta is empty.